ModelReplacementAPI/ModelReplacementAPI_Plugin.cs

```diff
@@ -285,8 +282,6 @@ public static void ManageRegistryBodyReplacements(ref PlayerControllerB __instan
             {
                 try
                 {
-                    if (__instance.playerSteamId == 0) { return; }
```

Comment:
I tested and for me steam id was always 0 in LAN mode, whether I launched the game from steam or not.